### DISCUSSION TITLE
Error: 500: //developers.google.com/identity/sign-in/web/devconsole-project

### DISCUSSION THREAD

**Discussion by distinctperspective** [AntlerVC/firetable#1011] [Support / Q&A] [answered]
👉 **Please describe the steps that you took that led to this bug.**

500: //developers.google.com/identity/sign-in/web/devconsole-project. Raw server response: "{"error":{"code":401,"message":"Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.","errors":[{"message":"Invalid Credentials","domain":"global","reason":"authError","location":"Authorization","locationType":"header"}],"status":"UNAUTHENTICATED","details":[{"@type":"type.googleapis.com/google.rpc.ErrorInfo","reason":"ACCOUNT_STATE_INVALID","domain":"googleapis.com","metadata":{"method":"google.cloud.identitytoolkit.v1.AccountManagementService.GetAccountInfo","email":"rowy-service@aymm-2022-12-02-flutterflow.iam.gserviceaccount.com","service":"identitytoolkit.googleapis.com"}}]}}"

When I tried to connect to my firebase

**Comment by harinij** [comment, marked as answer]
Hi @distinctperspective, just want to make sure you were logged into the account with the same email that has the firebase project permission. It is a bit hard to debug this error from the above message and we would need more information. If convenient you can book a [short call](http://rowy.io/meet) with us to get this resolved.
